django/db/backends/base/base.py

```diff
@@ -238,11 +241,15 @@ def _cursor(self, name=None):
 
     def _commit(self):
         if self.connection is not None:
+            if self.queries_logged:
+                logger.debug("COMMIT")
             with self.wrap_database_errors:
                 return self.connection.commit()
 
     def _rollback(self):
         if self.connection is not None:
+            if self.queries_logged:
+                logger.debug("ROLLBACK")
```

Comment:
This should be moved to the `CursorDebugWrapper`.

Reply:
This should be logged also in `self.queries_log` with the time taken to execute. I think we can add a context manager similar to the `debug_sql` but to the `BaseDatabaseWrapper`.